### PR TITLE
Update the styling of the WooPayments banner for the mobile version

### LINKED ISSUE #46231: [GlobalStep] "WooPayment" banner fails to render properly on "Woocommerce > Settings > Payment" page for smaller viewports.
### Prerequisites

- [ ] I have carried out troubleshooting steps and I believe I have found a bug.
- [ ] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

"WooPayment" banner fails to render properly on "Woocommerce > Settings > Payment" page for smaller viewports.

### Expected behavior

"WooPayment" banner should render properly on "Woocommerce > Settings > Payment" page for smaller viewports.

### Actual behavior

"WooPayment" banner fails to render properly on "Woocommerce > Settings > Payment" page for smaller viewports.

### Steps to reproduce

1. Create a test site.
2. Install and activate the Woocommerce version 8.8.0.40 plugins.
3. Install and activate WooPayment plugin.
4. Go to "Woocommerce > Settings > Payment" page.
5. Open browser console window and set viewport to mobile device.
6. Observe that, "WooPayment" banner fails to render properly for smaller viewports.

### Note: 
Issue occurs with the 8.8.0.40 version (but not on 8.8 RC) so it may persist in the 8.9 release if not addressed.

### Screenshot:

image (4)

### WordPress Environment

Woocommerce Version: 8.8.0.40
WordPress version:6.5

PC:
Windows 10
Chrome (Version 123.0.6312.106)
Firefox (Version 124.0.2)

### Isolating the problem

- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.
<details>

`
### WordPress Environment ###
WC Version: 8.8.0.40
REST API Version: ✔ 8.8.0.40
Action Scheduler Version: ✔ 3.7.3
Log Directory Writable: ✔
WP Version: 6.5
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: nginx
PHP Version: 8.1.27
PHP Post Max Size: 2 GB
PHP Time Limit: 1200
PHP Max Input Vars: 6144
cURL Version: 8.4.0
OpenSSL/3.0.11

SUHOSIN Installed: –
MySQL Version: 10.6.17-MariaDB-log
Max Upload Size: 2 GB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 8.8.0.40
WC Database Prefix: wp_
Total Database Size: 3.12MB
Database Data Size: 1.42MB
Database Index Size: 1.70MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 0.14MB + Index: 0.03MB + Engine InnoDB
wp_postmeta: Data: 0.11MB + Index: 0.06MB + Engine InnoDB
wp_posts: Data: 0.16MB + Index: 0.06MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_orders: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
wp_wc_orders_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_addresses: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_operational_data: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

attachment: 24
page: 7
post: 14
product: 21
product_variation: 7
revision: 2
shop_order_placehold: 1
wp_font_face: 33
wp_font_family: 12
wp_global_styles: 1
wp_navigation: 1
wp_template_part: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (5) ###

Query Monitor: by John Blackbourn – 3.15.0
Akismet Anti-spam: Spam Protection: by Automattic - Anti-spam Team – 5.3.2
Companion Plugin: by Osk – 1.30
WooPayments: by Automattic – 7.4.0
WooCommerce: by Automattic – 8.8.0.40

### Inactive Plugins (1) ###

Jetpack: by Automattic – 13.3-beta

### Dropin Plugins () ###

advanced-cache.php: advanced-cache.php
db.php: Query Monitor Database Class (Drop-in)
object-cache.php: Memcached

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to Woo.com: –
Enforce Approved Product Download Directories: ✔
HPOS feature screen enabled: ✔
HPOS feature enabled: ✔
Order datastore: Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore
HPOS data sync enabled: –

### Logging ###

Enabled: ✔
Handler: Automattic\WooCommerce\Internal\Admin\Logging\LogHandlerFileV2
Retention period: 30 days
Level threshold: –
Log directory size: 7 KB

### WC Pages ###

Shop base: #7 - /shop/
Cart: #8 - /cart/
Checkout: #9 - /checkout/
My account: #10 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Twenty-Four
Version: 1.1
Author URL: https://wordpress.org
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ❌ Not declared

### Templates ###

Overrides: /srv/htdocs/wp-content/plugins/woocommerce/templates/block-notices/error.php
/srv/htdocs/wp-content/plugins/woocommerce/templates/block-notices/notice.php
/srv/htdocs/wp-content/plugins/woocommerce/templates/block-notices/success.php

### WooPayments ###

Version: 7.4.0
Connected to WPCOM: ❌ No
Logging: Disabled

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customize-store
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
product-variation-management
product-virtual-downloadable
product-external-affiliate
product-grouped
product-linked
product-pre-publish-modal
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: minified-js
new-product-management-experience
product-custom-fields
settings
async-product-editor-category-field
launch-your-store

Daily Cron: ✔ Next scheduled: 2024-04-05 07:02:32 +00:00
Options: ✔
Notes: 63
Onboarding: skipped

### Action Scheduler ###

Complete: 40
Oldest: 2024-04-04 07:02:39 +0000
Newest: 2024-04-04 08:15:19 +0000

Failed: 1
Oldest: 2024-04-04 07:08:31 +0000
Newest: 2024-04-04 07:08:31 +0000

Pending: 2
Oldest: 2024-04-04 19:02:39 +0000
Newest: 2024-04-05 07:02:39 +0000

### Status report information ###

Generated at: 2024-04-04 14:55:52 +00:00
`

</details>

### PR DESCRIPTION
**Pull request by mordeth**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:
Fixes: https://github.com/woocommerce/woocommerce/issues/46231

Fix styling issues with WooPayments banner on mobile version.

### How to test the changes in this Pull Request:
1. Apply [this patch](https://gist.github.com/oaratovskyi/672dd55710bb8899a516d914276ea02f) locally
2. Build the changes
3. Go to "Woocommerce > Settings > Payment" page.
4. Open browser console window and set viewport to mobile device.
5. Observe that, "WooPayment" banner renders properly for smaller viewports.

| Before | After |
|---|---|
| image | image | 

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [x] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message
Fix styling issues with WooPayments banner on mobile version.

#### Comment 

</details>

### CODE REVIEW

**Review by oaratovskyi — APPROVED** [review]
Thanks Cvetan!
Code looks good and works well for small viewports 🟢

File: packages/js/onboarding/src/components/WCPayBanner/WCPayBanner.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .woocommerce-recommended-payments-banner {
 	margin: 0 15px 10px 0;
 	animation: isLoaded;
+	min-width: 550px;
```
Comment (anu-rock) [inline packages/js/onboarding/src/components/WCPayBanner/WCPayBanner.scss]: Noting that this was earlier `750px` before being removed in https://github.com/woocommerce/woocommerce/pull/46003/files#diff-7b7f0b500af945ff65443ee6f1dfb5e54091a530ff2fbd1b7886724328f529fe. I haven't manually tested this branch, but it seems 550 is enough?

File: packages/js/onboarding/changelog/46647-fix-wcpay-banner-mobile-screen [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Fix styling issues with WooPayments banner on mobile version.
```
Comment (anu-rock) [inline packages/js/onboarding/changelog/46647-fix-wcpay-banner-mobile-screen:4]:
```suggestion
Fix styling issues in the WooPayments banner on mobile-sized viewports.
```

File: packages/js/onboarding/src/components/WCPayBanner/WCPayBanner.scss (hunk 1 above)
Comment (mordeth) [reply packages/js/onboarding/src/components/WCPayBanner/WCPayBanner.scss]: Indeed, it was initially set to `750px`, but based on my tests, it pushes the banner beyond the visible area on iPhone SE. `550px` seems to be the optimal size that accommodates all viewports.

File: packages/js/onboarding/changelog/46647-fix-wcpay-banner-mobile-screen (hunk 2 above)
Comment (mordeth) [reply packages/js/onboarding/changelog/46647-fix-wcpay-banner-mobile-screen:4]: Your suggestion seems better, I've updated it!

**Comment by rodelgc** [comment]
Removing the `needs: external testing` label for now, since this PR is still open, and hasn't undergone PR testing analysis yet.

**Pull request merged**